mozilla-mobile/android-components/components/ui/tabcounter/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="mozac_ui_tabcounter_description">Tab Counter</string>
```

Comment:
This really needs a localization comment (I guess it's the accessibility text for the number of open tabs).